jobs/config_sort.py

```diff
@@ -34,5 +50,32 @@ def sort():
         fp.write('\n')
         fp.truncate()
 
+
+def sorted_prow_config():
```

Comment:
I don't think it should live in `prow/`, since prow doesn't care if they're sorted or not.